sync2jira/intermediary.py

```diff
@@ -299,6 +299,8 @@ def matcher(content: Optional[str], comments: list[dict[str, str]]) -> str:
     """
 
     def find_it(input_str: str) -> str:
+        if input_str is None:
+            return None
```

Comment:
Looks like a good idea, although, if we're gonna do that, we could go one step further:
```suggestion
        if not input_str:
            return None
```
which will skip the search on empty inputs as well as `None` inputs.

Reply:
Heh, that `not input_str` was my first impulse, but I went with `input_str is None` thinking "explicit is better than implicit". I'm looking for None here, let me be clear about that.

But, either way. :)